simplq/src/components/common/QrScanner/QrScanner.js

```diff
@@ -0,0 +1,77 @@
+//Component responsible for the /scanQr route
+
+import React, { Component } from 'react';
+import QrReader from 'react-qr-scanner';
+import styles from "./QrScanner.module.scss";
+
+class QRscanner extends Component {
+  constructor(props){
```

Comment:
Could you convert this to a function component too? This is the only one left.

Reply:
Yes, I have done it in my latest commit